### ISSUE TITLE
Mesa Intel(R) UHD Graphics 615 (AML-KBL) not supported?

### ISSUE THREAD

**Issue by sarigama** [warpdotdev/Warp#4385]
### Describe the bug

tried to run both .deb .appimage ver on my chromebook and both bumped into:

virgl(Mesa Intel(R) UHD Graphics 615 (AML-KBL))
Driver: Unknown

### To reproduce

clicking the icon doesnt do anything .. launching it from terminator(terminal) says what is in the attachment

### Expected behavior

simply doesnt launch and complains about config, fonts, and the driver ..

### Screenshots

![warperrot](https://github.com/warpdotdev/Warp/assets/1692100/25b46556-b0c7-4905-8945-6cb02d52eb5d)

### Operating system

Linux

### Operating system and version

chromeos121

### Regression

No, this bug or issue has existed throughout my experience using Warp

### Does this block you from using Warp daily?

Yes, this issue prevents me from using Warp daily.

### Is this a Warp specific issue? (i.e. does it happen in Terminal, iTerm, Kitty, etc.)

Yes, this I confirmed this only happens in Warp, not other terminals.

### Warp Internal (ignore): linear-label:b9d78064-c89e-4973-b153-5178a31ee54e

None

**Comment by Project-42** [comment]
Hi, try this version and see if works for you.. it does for me (also using chromebook)
https://github.com/warpdotdev/Warp/issues/4284#issuecomment-1967454359

**Comment by LFd3v** [comment]
It seems that OP is trying to use Warp inside a virtual machine because the detect GPU is `virgl`. Does it has the [graphics requirements](https://docs.warp.dev/getting-started/getting-started-with-warp#linux)?

**Comment by LFd3v** [comment]
Also, maybe forcing OpenGL mode will work, as it seems to [work with VirtualBox](https://github.com/warpdotdev/Warp/issues/4373#issuecomment-1973363946).

**Comment by Project-42** [comment]
> It seems that OP is trying to use Warp inside a virtual machine because the detect GPU is `virgl`. Does it has the [graphics requirements](https://docs.warp.dev/getting-started/getting-started-with-warp#linux)?

Yes, if OP is testing like me, is using Crostini -> https://chromium.googlesource.com/chromiumos/docs/+/master/containers_and_vms.md#:~:text=Crostini%20is%20the%20umbrella%20term,the%20default%20first%2Dparty%20experience.

**Comment by alokedesai** [comment]
@sarigama @Project-42 Ping on this. Are you still able to reproduce this? Thank you!
